app/helpers/date_time_helper.rb

```diff
@@ -8,4 +8,23 @@ def date_time(date, format: nil)
 
     I18n.l(date, format: :short)
   end
+
+  def time_based_heading
```

Comment:
They are one second off, but since this is the last when statement, it wouldn't matter cause it would never reach this part of the statement if it is midnight or later. Since it removes a line of code, I like it! thanks :wink: